osu.Game/Rulesets/Objects/Drawables/DrawableHitObject.cs

```diff
@@ -186,13 +187,16 @@ private void updateState(ArmedState newState, bool force = false)
         /// <summary>
         /// Apply (generally fade-in) transforms leading into the <see cref="HitObject"/> start time.
         /// The local drawable hierarchy is recursively delayed to <see cref="LifetimeStart"/> for convenience.
+        ///
+        /// By default this will fade in the object from zero with no duration.
         /// </summary>
         /// <remarks>
         /// This is called once before every <see cref="UpdateStateTransforms"/>. This is to ensure a good state in the case
         /// the <see cref="JudgementResult.TimeOffset"/> was negative and potentially altered the pre-hit transforms.
         /// </remarks>
         protected virtual void UpdateInitialTransforms()
```

Comment:
```suggestion
         protected virtual void UpdateInitialTransforms() => this.FadeInFromZero();
```

Reply:
either or. if we wanted to enforce this we could do it via r# rules.

Reply:
there are functions that just pass an argument to a field like:
```csharp
[BackgroundDependencyLoader]
private void load(Dependency dependency)
{
    this.dependency = dependency;
}
```
and it looks better with a code block so let's not do that